Hello guys

jucora opened the pull request:
Hello guys.

I'm happy to present my TODO LIST PROJECT :)

I've been trying to fix Linter issues with FONTAWESOME and DATE-FNS: 

Unable to resolve path to module '@fortawesome/fontawesome-free/js/fontawesome'. (import/no-unresolved)

Unable to resolve path to module 'date-fns'. (import/no-unresolved)

Do you know how can I fix this?


Thanks for code review

Have a nice day!

jucora commented:
Hello guys...

I'm learning about SOLID, I consider this topic very deep, I tried my best to apply the single responsability principle. As you can see, every method in my app only make one task, I also added comments to my code for reviewers to make my code more clear.

Please let me know what do you think and if there is something I should improve.


Thanks a lot

fatymahmed commented on .eslintrc.json:
```diff
@@ -0,0 +1,19 @@
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "import/no-unresolved": "off"
+  }
```
- Don't change linter config files

jucora commented:
Hello Fatima and thanks for your review.

The reason that I modified the linter config is because I'm usinf font awesome in my app, and there is not a real solution to fix IMPORTS FONTAWESOME ISSUES, in order to follow your review, I added the line // eslint-disable-next-line import/no-unresolved to every font awesome import instead of change it on linter config file, please let me know if that is ok because I tried a lot of ways and did not work, I tried to install import resolve plugins and more but I still got the error, that is why I did that.

I also made all changes require, like adding .DS_Store to gitignore and also I added some instructions in my readme.

For the live version link, I will added it after my project is good to go because I use github pages tool and it only allows me to assign a live link in the master branch, so after my project is approved I'll add the link for live version.

Thanks :)
